lib/ex_sdp/media.ex

```diff
@@ -205,17 +206,17 @@ end
 
 defimpl String.Chars, for: ExSDP.Media do
   def to_string(media) do
-    import ExSDP.Sigil
```

Comment:
Now when I'm looking at this I realize it was a good idea to have a separate module for the sigil :P but you can leave that as is ;)